### ISSUE TITLE
Support for multiple folders doesn't seem to be working as advertized

### ISSUE THREAD

**Issue by Neurrone** [sentriz/gonic#189]
Using the docker image v0.14, I see this log upon startup:

```
gonic    | 2022/02/02 20:45:52     music-path      /audiobooks                                                                       
```

Even though I have the `GONIC_MUSIC_PATH` variable set to `/music,/audiobooks`.

Using play:sub, I also only see the music folder to filter by.

**Comment by Neurrone** [comment]
Ok on a second read, I realize that the readme says version 0.15+ but the latest version released is still only 0.14.

When will 0.15 be released?

In the meantime, I'll pull a newer Docker build to test this.

**Comment by sentriz** [comment]
0.15 should be released shortly :+1: you can follow the progress here https://github.com/sentriz/gonic/pull/160
